modules/serverless-installing-cli-linux-ibm-power-rpm.adoc

```diff
@@ -0,0 +1,43 @@
+[id="installing-cli-linux-ibm-power-rpm_{context}"]
+= Installing the Knative CLI for Linux on IBM Power Systems using an RPM
+
+For {op-system-base-full}, you can install the `kn` CLI as an RPM if you have an active {product-title} subscription on your Red Hat account.
+
+.Procedure
+
+. Register with a subscription service during the firstboot process:
++
+[source,terminal]
+----
+# subscription-manager register
+----
+
+. Refresh the subscription-manager:
++
+[source,terminal]
+----
+# subscription-manager refresh
+----
+
+. Attach the subscriptions to a system by specifying ID of the subscription pool, using the --pool option:
++
+[source,terminal]
+----
+# subscription-manager attach --pool=<pool_id> <1>
+----
++
+<1> Pool ID for an active {product-title} subscription
+
+. Enable the repository using Red Hat Subscription Manager:
++
+[source,terminal]
+----
+# subscription-manager repos --enable="openshift-serverless-1-for-rhel-8-ppc64le-rpms"
+----
+
+. Install the openshift-serverless-clients on the system:
```

Comment:
`openshift-serverless-clients` formatting, also what are openshift-serverless-clients, is this a resource? I'd call that out